### PR TITLE
exbin

### PR DESCRIPTION
**Pull request by uziwuzzy**
implement exbin on credit card

### CODE REVIEW

File: MidtransDemo/MidtransDemo/MDOrderViewController.m [hunk 1]
```diff
@@ -107,7 +107,7 @@ - (void)viewDidLoad {
     
     CONFIG.callbackSchemeURL = @"demo.midtrans://";
     CONFIG.shopeePayCallbackURL = @"demo.midtrans://";
-    CONFIG.uobCallbackURL = @"demo.midtrans://";
+//    CONFIG.uobCallbackURL = @"demo.midtrans://";
```
Comment (sulisyo) [inline MidtransDemo/MidtransDemo/MDOrderViewController.m:110]: did you forgot to re-enable this?

Comment (uziwuzzy) [reply MidtransDemo/MidtransDemo/MDOrderViewController.m]: my bad, Done pak.

File: MidtransCoreKit/MidtransCoreKit/Core Api/Exbin/MIDExbinData.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//
+//  MIDExbinData.h
```
[the remark this answers is not in the document]
Comment (uziwuzzy) [reply MidtransCoreKit/MidtransCoreKit/Core Api/Exbin/MIDExbinData.h:2]: it's generated automatically by xcode.

**Review by askur87 — APPROVED** [review]
LGTM

**Pull request merged**